### PR TITLE
Update Angular docs and samples after removing 'divarea' editor type

### PR DESCRIPTION
**Pull request by Dumluregn**
Docs update for https://github.com/ckeditor/ckeditor4-angular/issues/130.

Should be merged when https://github.com/ckeditor/ckeditor4-angular/issues/133 will be released.

Also closes https://github.com/ckeditor/ckeditor4/issues/4125 and partially https://github.com/ckeditor/ckeditor4/issues/3821.

### CODE REVIEW

**Comment by f1ames** [comment]
Ok, so let's wait until Angular integration `2.0.0` is on NPM (hopefully this/next week) and then we can get back to this PR :hourglass\_flowing\_sand:

**Comment by f1ames** [comment]
> Ok, so let's wait until Angular integration `2.0.0` is on NPM (hopefully this/next week) and then we can get back to this PR

`2.0.0` integration version is not on NPM yet but it doesn't block us from reviewing this PR. The only thing to check before merging is if examples works with latest version.

**Review by f1ames — APPROVED** [review]
LGTM :+1:

Added some more info on `divarea` usage in `2.0.0` and earlier versions.

**Review by godai78 — APPROVED** [review]
Minor fixes only.

File: docs/guide/dev/integration/angular/README.md [hunk 1]
```diff
@@ -77,14 +77,28 @@ By default, starting from stable `1.0.0` version, the CKEditor 4 Angular compone
 ></ckeditor>
 ```
 
-So there are 3 possible values for the `type` property:
+### Using Div Area Editor Type
```
Comment (AnnaTomanek) [inline docs/guide/dev/integration/angular/README.md:80]:
```suggestion
### Using the Div-based Editor Type
```

File: docs/guide/dev/integration/angular/README.md [hunk 2]
```diff
@@ -77,14 +77,28 @@ By default, starting from stable `1.0.0` version, the CKEditor 4 Angular compone
...
 
-* `classic` (default)
-* `divarea`
-* `inline`
+Before the [`2.0.0` integration version](https://www.npmjs.com/package/ckeditor4-angular/v/2.0.0) there was also a separate `divarea` editor type (an {@link guide/dev/inline/README inline editor} with a {@link features/uitypes/README#fixed-ui-for-inline-editor fixed UI}).
 
-Notes:
-* The [Div Editing Area plugin](https://ckeditor.com/cke4/addon/divarea) must be {@link guide/dev/plugins/README included in your editor build} to use `divarea` editor type. In this case, however, there is no need to list it in the {@linkapi CKEDITOR.config#plugins `config.plugins`} or {@linkapi CKEDITOR.config#extraPlugins `config.extraPlugins`} options as the Angular component does this automatically for you.
+To use the `divarea` editor in `2.0.0` and later versions you just need to include a [Div Editing Area plugin](https://ckeditor.com/cke4/addon/divarea) in the editor config:
```
Comment (AnnaTomanek) [inline docs/guide/dev/integration/angular/README.md:84]:
```suggestion
To use the `divarea` editor in `2.0.0` and later versions, you just need to include the [Div Editing Area](https://ckeditor.com/cke4/addon/divarea) plugin in the editor configuration:
```

File: docs/guide/dev/integration/angular/README.md [hunk 3]
```diff
@@ -77,14 +77,28 @@ By default, starting from stable `1.0.0` version, the CKEditor 4 Angular compone
...
+
+```html
+<ckeditor
+	data="<p>Some initial data</p>"
+	type="inline"
+	[config]="{ extraPlugins: 'divarea' }">
+></ckeditor>
+```
+
+To use the `divarea` editor in versions prior `2.0.0` (up to `1.3.0`), you have to change editor type to `divarea`:
```
Comment (AnnaTomanek) [inline docs/guide/dev/integration/angular/README.md:94]:
```suggestion
To use the `divarea` editor in versions prior `2.0.0` (up to `1.3.0`), you have to change the editor type to `divarea`:
```

File: docs/sdk/examples/angular.html [hunk 4]
```diff
@@ -194,266 +194,229 @@ <h1>CKEditor 4 Angular Integration <a class="documentation" href="https://ckedit
...
+				<span *ngIf="event.counter > 1"> ({{ event.counter }})</span>
+			</li>
+		</ul>
+
+		<button (click)="clearEventsLog()">Clear events log</button>
+
+		<h3>Angular directives</h3>
+
+		<p><label><input type="checkbox" [(ngModel)]="isHidden">Hide the editor with the <code>[hidden]</code> directive.</label></p>
+		<p><label><input type="checkbox" [(ngModel)]="isRemoved">Remove the editor from DOM with the <code>*ngIf</code> directive.</label></p>
```
Comment (AnnaTomanek) [inline docs/sdk/examples/angular.html:278]:
```suggestion
		<p><label><input type="checkbox" [(ngModel)]="isRemoved">Remove the editor from the DOM with the <code>*ngIf</code> directive.</label></p>
```

**Review by AnnaTomanek — APPROVED** [review]
Minor suggestions added, looks fine otherwise! :+1:

**Comment by f1ames** [comment]
Rebased onto latets `stable`.

**Pull request merged**